### ISSUE TITLE
Update readme regarding the new TF-FGB device

### ISSUE THREAD

**Issue by fruggy83** [fruggy83/openocean#34]
Hi @dominikkv, kannst du vielleicht noch die readme zum Binding um ein paar Worte bzgl. TF-FGB erweitern? Vielen Dank im Voraus.

Gruß
Daniel

**Comment by dominikkv** [comment]
Ich habe die readme etwas ergänzt, Geräte & Channels hinzugefügt. Was genau kann ich bzgl. TF-FGB ergänzen?

**Comment by fruggy83** [comment]
Hi @dominikkv, dein Update sieht gut aus. Der TF-FGB ist nun gut dokumentiert. Mir ist nur aufgefallen, dass du einen ```batteryStorage``` channel erwähnst. Ist der noch aktuell?

**Comment by dominikkv** [comment]
Ja, der `batteryStorage` gibt mir Rätsel auf. Es geht hier um den Eltako TF-FKB, der ein unbenanntes A5 Status EEP sendet. Die Bedienungsanleitung definiert hier zwei Felder, `Batterie Spannung` und `Batterie Speicher`. Ich werde beim Support nachfragen, sobald ich aus dem Urlaub zurück bin.
